src/pkg/packaging-tools/sharedFramework/sharedFramework.csproj

```diff
@@ -0,0 +1,153 @@
+<Project Sdk="Microsoft.NET.Sdk" InitialTargets="GetPackageReference" DefaultTargets="Restore;Publish">
+  <!--
+    This project lays out a shared framework using the SDK targets.
+  -->
+
+  <UsingTask TaskName="ProcessSharedFrameworkDeps" AssemblyFile="$(LocalBuildToolsTaskDir)core-setup.tasks.dll" />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <NETCoreAppMaximumVersion>3.0</NETCoreAppMaximumVersion>
+    <RuntimeIdentifier>$(PackageRID)</RuntimeIdentifier>
+    <PackageId>unused</PackageId>
+
+    <ProjectDepsFileName>$(SharedFrameworkName).deps.json</ProjectDepsFileName>
+
+    <MicrosoftNETPlatformLibrary>do-not-consider-existing-platform</MicrosoftNETPlatformLibrary>
+
+    <!-- Don't reference NETCore.App -->
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+    <!-- Only Restore from our built packages -->
+    <RestoreSources>$(PackageOutputPath)</RestoreSources>
+    <RestoreConfigFile>$(ProjectDir)NuGet.config</RestoreConfigFile>
```

Comment:
Why are you setting config file?  In dotnet-trusted this was a config file that cleared everything.

Reply:
I'll revisit this, doesn't seem necessary with `RestoreSources`.

Reply:
I think when I was playing with this before I added stuff until I saw what I expected in the `project` section of the project.assets.json.  This lists all the places NuGet considered when doing the restore and we need to make sure they are all locked down.

Reply:
Removing this line does this (and only this) in the `project.assets.json`, `project` section, and nothing to the sfx contents:

```diff
       "configFilePaths": [
-        "C:\\git\\core-setup\\NuGet.config"
+        "C:\\git\\core-setup\\NuGet.Config",
+        "C:\\Users\\dagood\\AppData\\Roaming\\NuGet\\NuGet.Config",
+        "C:\\Program Files (x86)\\NuGet\\Config\\Microsoft.VisualStudio.Offline.config"
       ],
```

I figure we might as well keep it in case there's some weird stuff the chain of NuGet.Configs might be able to do. I'll add a comment that it isn't doing anything to `Only Restore from our built packages` though.